src/layout/Footer/FooterDesktop.tsx

```diff
@@ -51,6 +61,8 @@ export const FooterDesktop = () => {
           label: stringGetter({ key: STRING_KEYS.DEGRADED }),
         };
 
+  const openHelpDialog = useCallback(() => dispatch(openDialog(DialogTypes.Help())), [dispatch]);
```

Comment:
the usecallback isn't really necessary here, doesn't get us anything since footerbutton isn't memo